packages/@uppy/transloadit/src/Assembly.ts

```diff
@@ -233,6 +250,10 @@ class TransloaditAssembly extends Emitter {
     Object.keys(next.uploads)
       .filter((upload) => !has(prev.uploads, upload))
       .forEach((upload) => {
+        // TODO: what should this be?
+        // We can't use a random string to index an array.
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore this is a mistake
         this.emit('upload', next.uploads[upload])
```

Comment:
Well technically we could, but of course TS is not going to like this. I don't think it's a random string though, since it's returned by `Object.keys`

Reply:
Rewrote it to be more sane.